docs/UserGuide.md

```diff
@@ -528,9 +528,13 @@ Lists all incomes with the category `prize` and/or `gift`, with amounts between
 
 ### 4.5 Bookmark Expense
 
-Fine<span>$</span><span>$</span>e's Bookmark Expense feature is used to store frequent expenses that the user makes such as paying of monthly phone bills or buying bubble tea weekly.
+Fine\\$\\$e's Bookmark Expense feature is used to store frequent expenses that the user makes such as paying of monthly phone bills or buying bubble tea weekly.
 The user will be then be able to edit, delete and convert a bookmarked expense to conveniently add it into Fine<span>$</span><span>$</span>e's expense list.
 
+![Bookmark Expense Overview](images/userguide/bookmark/AnnotatedBookmarkExpenseOverview.png)
+
+Bookmark Expenses List Panel in the Expenses Tab
```

Comment:
This line isn't very clear as to what it's for. :/

![image](https://user-images.githubusercontent.com/5585517/97747033-44afa100-1b26-11eb-8922-677e59839d75.png)

If it is meant to be captions to the image, perhaps we could work out some styling to use that makes its purpose more obvious?

Same for the other captions.

Reply:
To be resolved in a separate PR. Issue #256 opened.